 design/README.MD

```diff
@@ -0,0 +1,456 @@
+## Overview
+
+Smithy is an interface definition language.
+It lets you abstract your interface
+from any given executable implementation.
+By focusing on the abstract interface
+Smithy helps you build an interface
+that can safely evolve
+maintaining both forward and backwards compatibility.
+
+Dafny is a programming language with formal verification features.
+Dafny code is translated to higher-level languages
+to provide interoperability for your verified code
+like Java or C#.
```

Comment:
> Dafny code is translated to higher-level languages
to provide interoperability for your verified code
like Java or C#.

seems a bit confusingly worded. maybe:

> Dafny code is translated to higher-level languages like Java or C#
to provide interoperability with your verified code

?